assets/js/components/CookieOverlay.js

```diff
@@ -23,8 +24,16 @@ module.exports = class CookieOverlay {
     this.doc.querySelector('body').appendChild(this.$overlay);
   }
 
-  static previouslyAccepted(doc) {
-    return !!utils.getCookieValue('cookieNotificationAccepted', doc.cookie);
+  previouslyAccepted() {
+    return !!utils.getCookieValue('cookieNotificationAccepted', this.doc.cookie);
+  }
+
+  accept() {
+    this.$overlay.parentNode.removeChild(this.$overlay);
+    const expiryDate = 'Tue, 19 January 2038 03:14:07 UTC';
+    const domain = this.window.elifeConfig.domain ? this.window.elifeConfig.domain : 'elifesciences.org';
```

Comment:
As `elifeConfig` is set externally, access to its properties should be wrapped in a `try/catch` in case it's not available for some reason. Additionally, logging to New Relic could be helpful, for an example of this see the catch block at https://github.com/elifesciences/pattern-library/blob/e4e323666e56701d0f06437841dea275a4c8f333/assets/js/elife-loader.js#L25